src/java.base/share/classes/java/util/Spliterator.java

```diff
@@ -304,7 +304,7 @@
      * Subsequent behavior of a spliterator is unspecified if the action throws
      * an exception.
      *
-     * @param action The action
+     * @param action The action which will be invoked at-most once
```

Comment:
If you want to align with the language specified in `Consumer` i recommend:
```suggestion
     * @param action The action whose operation is performed at-most once
```